src/util/internalPeaks/getInternalPeaks.ts

```diff
@@ -134,3 +132,15 @@ function getNormalizedValue(
   }
   return value;
 }
+
+function getParameterByKey<
+  Parameters extends Record<Parameter, InitialParameter>,
+>(
```

Comment:
Not sure why you created a function for it? It seems easier to understand if the optional chaining is inline.

Reply:
It is true, I did like it